client/src/components/reader/TocDrawer.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import { Toc } from 'components/reader';
+import ScrollLock from 'components/global/Utility/ScrollLock';
 import classNames from 'classnames';
 
 export default class TocDrawer extends Component {
```

Comment:
It looks like this component might be missing a displayName static property. Not your fault, but we may as well fix in this commit.